xaynetwork/xayn_discovery_app
# TB-3996 Track onboarding bottom sheet dismissed analytics events

### What 🕵️ 🔍

- Track onboarding bottom sheet dismissed analytics events.

----------

### How to test (please adjust template) 🥼 🔬
- Open app 
- Navigate to home
- dismiss onboarding bottom sheet
- open personal area screen
- dismiss onboarding bottom sheet
- check analytics

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-3996)

----------
